projects/cobbler-api/src/lib/cobbler-api.service.spec.ts

```diff
@@ -103,7 +103,7 @@ describe('CobblerApiService', () => {
       systems: '',
       buildisodir: '',
       distro: '',
-      standalone: false,
+      standalone: true,
```

Comment:
I know this is picky but please drop this change. It has nothing to do with the Angular standalone notation and is a domain-specific term.

In this scenario `standalone` means that the ISO that is produced will boot offline or if it requires a network connection to boot the installer.